ntcore/CMakeLists.txt

```diff
@@ -37,11 +37,7 @@ install(TARGETS ntcore EXPORT ntcore)
 install(DIRECTORY src/main/native/include/ DESTINATION "${include_dest}/ntcore")
 install(DIRECTORY ${WPILIB_BINARY_DIR}/ntcore/generated/main/native/include/ DESTINATION "${include_dest}/ntcore")
 
-if (WITH_FLAT_INSTALL)
-    set (ntcore_config_dir ${wpilib_dest})
-else()
-    set (ntcore_config_dir share/ntcore)
-endif()
+set(ntcore_config_dir share/ntcore)
```

Comment:
Should we remove these sets entirely now that the conditional is gone?  I think the result may be used only in the two install() lines?

Reply:
Yeah, sounds like a good idea.